CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,82 @@
+# Citizen Code of Conduct
+
+## 1. Purpose
+
+A primary goal of Kafka Go is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).
```

Comment:
I've seen "Kafka Go", "Kafka-Go", and "kafka-go" in various places. 😅  I believe the latter is the canonical version, so a search/replace is in order.

Reply:
kafka-go is the correct name indeed 👍

Reply:
Easy enough to change.